src/layout/components/NavHeader/index.vue

```diff
@@ -184,7 +184,7 @@ export default {
         }
 
         &:hover {
-          background-color: rgba(0, 0, 0, 0.12) !important;
+          background-color: rgba(0, 0, 0, 0.2);
         }
       }
 
```

Comment:
There aren't any immediate problems with this code. However, it should be updated to use CSS preprocessors like Sass for a more readable and maintainable stylesheet.

In summary:

- Update the styles using SCSS or SASS

Remember that I've provided an overview without looking over every line of the code due to its scope. Detailed review might reveal further issues.